TeamCode/src/main/java/org/firstinspires/ftc/teamcode/AprilTagDetectionFrameProcessor.java

```diff
@@ -83,31 +88,25 @@ public AprilTagDetectionPipeline(double tagsize, double fx, double fy, double cx
     }
 
     @Override
-    public void finalize()
-    {
+    protected void finalize() {
         // Might be null if createApriltagDetector() threw an exception
-        if(nativeApriltagPtr != 0)
-        {
+        if (nativeApriltagPtr != 0) {
```

Comment:
Also, I reformatted this whole file so it matches our project style, with open brace on the same line as the `if`.